src/main/java/mate/academy/hibernate/relations/service/impl/ActorServiceImpl.java

```diff
@@ -1,16 +1,28 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import java.util.Optional;
+import mate.academy.hibernate.relations.dao.ActorDao;
 import mate.academy.hibernate.relations.model.Actor;
 import mate.academy.hibernate.relations.service.ActorService;
 
 public class ActorServiceImpl implements ActorService {
+    private ActorDao actorDao;
+
+    public ActorServiceImpl(ActorDao actorDao) {
+        this.actorDao = actorDao;
+    }
+
     @Override
     public Actor add(Actor actor) {
-        return null;
+        return actorDao.add(actor);
     }
 
     @Override
     public Actor get(Long id) {
-        return null;
+        Optional<Actor> actor = actorDao.get(id);
+        if (actor.isPresent()) {
+            return actor.get();
+        }
+        throw new RuntimeException("Actor with id " + id + "not found");
```

Comment:
1. Structure can be simplified by using orElseThrow method
2. Better to use more precise exception class
```suggestion
        return actorDao.get(id).orElseThrow(
                () ->  new EntityNotFoundException("Actor with id " + id + "not found");
```